pkg/api/validation/validation_test.go

```diff
@@ -175,6 +188,10 @@ func TestValidateVolumes(t *testing.T) {
 			if errs[i].(*errors.ValidationError).Field != v.F {
 				t.Errorf("%s: expected errors to have field %s: %v", k, v.F, errs[i])
 			}
+			detail := errs[i].(*errors.ValidationError).Detail
+			if detail != "" && detail != "exactly 1 volume type is required" && detail != DnsLabelErrorMsg {
```

Comment:
how does the "exactly 1" error happen here?  It doesn't look like that is tested...